android/res/layout/edit_bookmark_common.xml

```diff
@@ -55,22 +54,23 @@
       android:background="?attr/selectableItemBackground"
       android:clickable="true"
       android:paddingTop="@dimen/margin_quarter_plus"
-      android:paddingBottom="@dimen/margin_half"
```

Comment:
Is this identifier used somewhere else? If not, please clean it up.

Reply:
This is used in a lot of other places
![image](https://user-images.githubusercontent.com/46300244/230200265-00a60066-8d8e-45de-b3c2-9b04c25b7c7d.png)